django_app/redbox_app/templates/support.html

```diff
@@ -1,22 +1,18 @@
-{% import "macros.html" as macros %}
-{% extends "base_generic_gov.html" %}
+{% extends "base.html" %}
 
-<!--TODO: Update variables-->
-
-{% block title %}
-  <title>Support - Redbox Copilot - GOV.UK</title>
-{% endblock %}
+{% set pageTitle = "Support" %}
 
 {% block content %}
   <h1 class="govuk-heading-l">Support</h1>
-  <span class="govuk-caption-l">Find out about the One Big Thing website, including how to sign in and use your <SYSTEM_NAME> account.</span>
-  <br>
+
+  {#
```

Comment:
Never seen this syntax before and I'm curios, what does it do?

Reply:
{# Anything inside here is a comment #}
Better (in some cases) than <!-- this comment --> as it won't be rendered as part of the HTML.
I just wanted to keep that as a reference incase we want to put anything similar back in.